frontend/app/(dashboard)/updates/company/page.tsx

```
@@ -72,29 +72,61 @@ const AdminList = () => {
+        ),
+      }),
+      columnHelper.accessor("sentAt", {
+        header: "Date",
+        size: 120,
+        minSize: 120,
+        maxSize: 120,
         cell: (info) => (
-          <Link href={`/updates/company/${info.row.original.id}/edit`} className="no-underline">
-            {info.getValue()}
-          </Link>
+          <div className="text-sm whitespace-nowrap">{info.getValue() ? formatDate(info.getValue() as Date) : "—"}</div>
```

ESLint

[error] Do not use any type assertions.

(@typescript-eslint/consistent-type-assertions)